README.md

```diff
@@ -71,6 +72,44 @@ Composite graph definitions can get long and cumbersome, so you can define Label
     vars.graphite_keys = ["{used,max}", "{currentThreadsBusy,currentThreadCount}"]
     vars.graphite_labels = ["Heap", "Threads"]
 
+### Area mode
+It is possible to fill the area below the graphed lines by adding the following to your Service definition.
+
+    vars.graphite.area_mode = "all" // default : "all"
+    vars.graphite.area_alpha = "0.1" // default : "0.1"
+
+To set it globally, customize the configuration file with :
+
+    graphite_area_mode = all
+    graphite_area_alpha = 0.1
+
+For details about the values, see doc here : http://graphite.readthedocs.io/en/latest/render_api.html?highlight=areaMode
+
+### Derivative graphs
+It is possible to plot derivative graphs instead of the measured values. This is useful for sensors giving increasing counters (like printer's page count, network IO bytes counter, ...). You can set it by adding the following to your Service definition.
+
+    vars.graphite.graph_type = "derivative" // default : "normal"
+
```

Comment:
A more flexible approach here might be to have a $target$ override. 

`vars.graphite.target = "summarize(nonNegativeDerivative($target$),'hour, 'sum')"`

Otherwise you can only set one derivative for all graphs.